misc_utilities/create_embedding.py

```diff
@@ -0,0 +1,154 @@
+# Imports
```

Comment:
This file is used mostly for dev and scratch work, and generated the embeddings used in the example invokes with embeddings in the testing_materials folder